doc/misc/npm-config.md

```diff
@@ -240,11 +240,16 @@ If true, then only prints color codes for tty file descriptors.
 
 ### depth
 
-* Default: Infinity
+* Default: -1
```

Comment:
Is there a way to do this without a magic number? This feels counterintuitive to me, and not really in keeping with the former range of `0..Infinity`.